File: bin/math/matdet.cpp
#include "commons/commons.h"
#include "maths/maths.h"

int main(int argc,char** args)
{
	int nM;
	std::vector<Matrix> matrices;
	Matrix m;
	cout << "Enter matrix :\n";
	cin >> m;
	matrices.push_back(m);
	cout << m.determinant() << "\n";
}

File: bin/math/.jav/MatDet.java
import maths.matrices.*;
import maths.Maths;
import commons.X;

public class MatDet
{
	public static void main(String[] args)throws Exception
	{
		SquareMatrix sm=new SquareMatrix(Matrix.inputMatrix());
		X.sopln(sm.getDeterminant());
	}
}
